### PR TITLE
TW 64 Updated information about public collaboration

### PR DESCRIPTION
**Pull request by HeatherStenson**
- Moves /docs/collaborating-in-postman/public-workspaces/ into /using-workspaces/
- Updates internal links to account for ^
- Reformats content in public-workspaces.md
- Adds additional information per comments from @gitThingsDone in TW-64

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+You can add existing collections and environments to another public workspace by sharing them.
+
+* To share a _collection_ to a public workspace, open it using Collections on the left side, then select the three dots icon <img alt="Three dots icon" src="https://assets.postman.com/postman-docs/icon-three-dots-v9.jpg" width="18px" style="vertical-align:middle;margin-bottom:5px"> > **Share collection**.
+* To share an _environment_ to a public workspace, open it using Environments on the left side, then select the three dots icon <img alt="Three dots icon" src="https://assets.postman.com/postman-docs/icon-three-dots-v9.jpg" width="18px" style="vertical-align:middle;margin-bottom:5px"> > **Share environment**.
+
+Select the public workspace you want to share to, and choose whether you also want to remove the element from its current workspace. Select **Share and Continue**.
+
+### Removing elements from a public workspace
+
+When you remove an element from a public workspace, it's no longer visible in that particular workspace. The element is still available in any other workspace in which it exists.
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:154]: I think these instructions need to be updated. Elements like collections and environments can no longer be in multiple workspaces, and rather can only exist in a single workspace. You could either move the collection/environment or delete it. The "Remove from workspace" option  is not available under the three dots menu.

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+### Editing public workspace details
+
+Users with an [Admin role](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles) for a workspace can edit workspace details, including updating the workspace's name, summary, description, and visibility. For more information, see [Editing workspace details](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#editing-workspace-details).
+
+> The [Public API Network](/docs/getting-started/exploring-public-api-network/) displays your workspace's name and summary. Make your workspace name and summary informative to improve its visibility on the Public API Network.
+
+### Adding elements to a public workspace
+
+You can add existing collections and environments to another public workspace by sharing them.
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:145]: I don't think this is true anymore, as elements can only exist in a single workspace now. Sharing a collection/environment just shares a link but doesn't make it available in another workspace. To do that, you would have to move the collection/environment.

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+
+> Deleting an element is different from removing it. When you delete an element, it's no longer available in any workspace. When you remove an element from a specific workspace, it will still be available in any other workspaces in which it exists.
+
+### Deleting a public workspace
+
+To delete a public workspace, you must revert its visibility. If you try to delete a public workspace by selecting the three dots icon <img alt="Three dots icon" src="https://assets.postman.com/postman-docs/icon-three-dots-v9.jpg" width="18px" style="vertical-align:middle;margin-bottom:5px"> on the right corner of the workspace overview, the option to delete is greyed out.
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:163]: Per Microsoft, change "is greyed out" to "Isn't available"
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/g/gray-grayed-out

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+
+![Deleting a public workspace](https://assets.postman.com/postman-docs/delete-option-greyed-public-workspace.jpg)
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:165]: Is this screenshot accurate? When I look at public workspaces, I don't see the three dots at upper right, but I do see "Workspace Settings"

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+
+To change the visibility of a public workspace and delete it:
+
+1. Change the visibility dropdown to **Team**, then **Save Changes**.
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:169]: Is this still accurate? When I look at a public workspace, I have to select "Workspace Settings" at upper right and then there's a delete button at the bottom

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+
+> Team members will receive an email and in-app notification when they're added to a public workspace.
+
+## Collaborating in another user's public workspace
+
+To collaborate with someone else in their public workspace, open the **Workspace** menu and select the workspace you want to work in.
+
+For **collections** and **environments**, [create a fork](/docs/collaborating-in-postman/version-control-for-collections/#forking-a-collection) and request to merge changes using a [pull request](/docs/collaborating-in-postman/version-control-for-collections/#creating-pull-requests).
+
+For **APIs**, navigate to the API and version. Select **Definition** > **Request Access** to request an editor role.
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:187]:
```suggestion
For **APIs**, navigate to the API and version. Select the **Definition** tab and then select **Request Access** to request an editor role.
```

Comment (mark-anthony-postman) [reply src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:187]: Per Microsoft, we can use ">" to connect actions that use the same selection method. But in this case you're selecting a tab and then a button, so I think it's best not to use ">"
https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui

**Review by mark-anthony-postman — APPROVED** [review]
Wow this is a huge improvement over the original. Much better organization and I love all the additional context and tips for making your workspace more engaging. I made some comments about some instructions that seemed out of date.

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md (hunk 5 above)
Comment (HeatherStenson) [reply src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:169]: Yipes, you're right! The old instructions were not up to date and I neglected to verify 😬  Will update!

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md (hunk 2 above)
Comment (HeatherStenson) [reply src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:145]: You're absolutely right! I should have gone through the instructions in the existing doc much more thoroughly. Trust but verify!

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md (hunk 1 above)
Comment (HeatherStenson) [reply src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:154]: Yikes, it's true, this is wildly out of date 🙈

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md (hunk 4 above)
Comment (HeatherStenson) [reply src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:165]: Updating!

**Review by jkonrath-postman — APPROVED** [review]
LGTM, except for Mark's stuff...

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+* [Collaborating in another user's public workspace](#collaborating-in-another-users-public-workspace)
+* [Next steps](#next-steps)
+
+## Enabling your team profile
+
+Before other Postman users can work with your team in a public workspace, your team needs to have a public profile. A public team profile will encourage others to collaborate with you in a public workspace, allow you to ask for contributions and gather feedback, and will help increase your API's discovery in search results. A public team profile will also show up on the [Public API Network](https://www.postman.com/explore).
+
+To make your [team profile](https://go.postman.co/settings/team/general) public:
+
+1. Select **Team** next to your avatar in the upper-right corner of Postman, then select **Team settings**.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:57]: Could consider using a > (mentioned [here](https://github.com/postmanlabs/postman-docs/blob/develop/DOCS_STYLE_GUIDE.md))

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md [hunk 8]
```diff
@@ -0,0 +1,199 @@
...
+
+> Within a team, you can't have two public workspaces with the same name.
+
+To convert an existing workspace to a public workspace:
+
+1. Open the workspace. In the **Overview** tab, select **Workspace Settings** in the upper right.
+1. Under **Visibility**, select **Public**.
+1. Select **Update**.
+
+> You must have an [Admin](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles) role for a workspace to change its visibility to public.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:93]: not sure on caps, but maybe something like

```suggestion
> You must be a [Workspace Admin](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles) to change the visibility of a workspace.
```

**Review by claireriley — APPROVED** [review]
LGTM! 🎉  Noted a few minor things to consider

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md [hunk 9]
```diff
@@ -0,0 +1,199 @@
...
+updated: 2022-03-11
+warning: false
+contextual_links:
+  - type: section
+    name: "Additional Resources"
+  - type: subtitle
+    name: "Related Blog Posts"
+  - type: link
+    name: "Creating workspaces"
+    url: "/docs/collaborating-in-postman/using-workspaces/creating-workspaces/"
```
[the remark this answers is not in the document]
Comment (HeatherStenson) [reply src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:14]: Good call! Will do.

**Review by gitThingsDone — APPROVED** [review]

**Pull request merged**